### ISSUE TITLE
Challenges test failure

### ISSUE THREAD

**Issue by SlimeDog** [BentoBoxWorld/Challenges#284]
Challenges b449, b450, b451

#### Describe the bug
```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   CompleteChallengeCommandTest.testSetup:206 expected:<bskyblock.c[omplete]> but was:<bskyblock.c[hallenges]>
[INFO] 
[ERROR] Tests run: 136, Failures: 1, Errors: 0, Skipped: 2
[INFO] 
[ERROR] There are test failures.
```

**Comment by tastybento** [comment]
Should be fixed by 95edafd

**Comment by SlimeDog** [comment]
Challenges b452 has no test error. Thanks.
